corehq/apps/translations/app_translations.py

```diff
@@ -196,6 +145,30 @@ def process_bulk_app_translation_upload(app, workbook):
     return msgs
 
 
+def _check_for_sheet_error(sheet, expected_sheets=Ellipsis, processed_sheets=Ellipsis):
```

Comment:
i'd just call this `validate_sheets` or `validate_workbook`.

Reply:
I like the symmetry of `_check_for_workbook_error`, `_check_for_sheet_error`, and `_check_for_sheet_warnings`